Developers/Newcomers_Guide.rst

```diff
@@ -17,6 +17,12 @@ because we care!
 
 **Congratulations!** You are now part of our community.
 
+.. note::
+    
+    The first thing you should do is check other developers' pull requests to get the 
+    hang of things.
```

Comment:
I wouldn't recommend this as the first thing, first make a PR and then take a look at others, not only looking but also helping. Learning together with the others.

IMO though.

CC @Adrianzatreanu @adtac